README-mux.md

```diff
@@ -0,0 +1,345 @@
+  * or use `ssh-copy-id -i /path/to/my/id_rsa.pub ...` to copy one over
+* enable passwordless sudo (if not using root above) e.g. in sudoers config:
+  * `centos ALL=(ALL) NOPASSWD:ALL`
+* allow connections on the following ports/protocols:
+  * icmp (for ping)
+  * tcp ports 22, 80, 443, 8443 (openshift console), 24284 (secure_forward)
+
+This will allow you to access the machine via ssh (in order to run Ansible -
+see below), to access the external services such as Kibana and mux, and to
+access the OpenShift UI console.
+
+ViaQ on OCP requires a RHEL and OCP subscription.
+
+ViaQ on Origin requires these [Yum Repos](#appendix-1-centos7-viaq-yum-repos).
+
+You will need to install the following packages: docker iptables-services NetworkManager
+
+You will need to configure sudo to not require a tty e.g. something like this:
+
+    # ls -l /etc/sudoers.d/999-cloud-init-requiretty
+    -r--r-----. 1 root root 21 Dec  1 13:17    /etc/sudoers.d/999-cloud-init-requiretty
+    # cat /etc/sudoers.d/999-cloud-init-requiretty
+    Defaults !requiretty
+
+You may need to disable SELinux `sudo setenforce Permissive` or edit
+`/etc/selinux/config` and use `SELINUX=permissive`
+
+You will need to ensure NetworkManager is running:
+
+    # systemctl daemon-reload
+    # systemctl enable NetworkManager
+    # systemctl start NetworkManager
+
+## Installing ViaQ
+
+These instructions and config files are for an all-in-one, single machine, run
+ansible on the same machine you are installing ViaQ on.
+
+*NOTE* *THIS IS NOT FOR PRODUCTION USE*
+
+The setup below is for a developer or demo machine, all-in-one, running
+Ansible in *local* mode to install ViaQ on the same machine as Ansible is
+running on.  It also configures the `AllowAllPasswordIdentityProvider` which
+means anyone can log in using the OpenShift UI.
+
+Ansible is used to install ViaQ and OCP or Origin using OpenShift Ansible.
+The following packages are required: openshift-ansible
+openshift-ansible-callback-plugins openshift-ansible-filter-plugins
+openshift-ansible-lookup-plugins openshift-ansible-playbooks
+openshift-ansible-roles
+
+    # yum install openshift-ansible \
+      openshift-ansible-callback-plugins openshift-ansible-filter-plugins \
+      openshift-ansible-lookup-plugins openshift-ansible-playbooks \
+      openshift-ansible-roles
+
+If the 3.5/1.5 versions of these packages are not available, you can use the
+git repo `https://github.com/openshift/openshift-ansible.git` and the
+`release-1.5` branch:
+
+    # git clone https://github.com/openshift/openshift-ansible.git -b release-1.5
+
+You will need to use the `ansible-playbook` command with an Ansible inventory
+file and a `vars.yaml` file.  You should not have to edit the inventory file.
+All customization can be done via the `vars.yaml` file.
+
+Download the files [vars.yaml.template](vars.yaml.template) and
+[ansible-inventory-origin-15-aio](ansible-inventory-origin-15-aio)
+
```

Comment:
It doesn't matter.  Should I just say `It doesn't matter where you save them, just remember the locations for the ansible-playbook command`?